src/modules/admin/markets/sagas/updateMarketSaga.ts

```diff
@@ -15,17 +16,33 @@ const enableMarketsConfig = (csrfToken?: string): RequestOptions => {
     };
 };
 
+const configUpdateOptions = (csrfToken?: string): RequestOptions => {
+    return {
+        apiVersion: 'sonic',
+        headers: { 'X-CSRF-Token': csrfToken },
+    };
+};
+
 export function* updateMarketSaga(action: MarketUpdateFetch) {
     try {
         yield all(action.payload.map(item => call(API.post(enableMarketsConfig(getCsrfToken())), '/admin/markets/update', item)));
+        if (action.callbackAction) {
+            const { scope, key, value, component } = action.callbackAction;
+            const payload = new FormData();
```

Comment:
you said on the call that this should be not a FormData

Reply:
Right now it's form data while BE takes FormData payload
I will send this changes to send JSON instead of FormData as separate PR